board/batocera/rockchip/rk3326/linux_patches/linux-006-2-4-mfd-Add-Rockchip-rk817-battery-charger-support.patch

```diff
@@ -0,0 +1,157 @@
+diff --git a/drivers/mfd/rk808.c b/drivers/mfd/rk808.c
```

Comment:
If you're going to be borrowing patches from mailing list use "mbox" button to download the patch to preserve authorship